### PR TITLE
line follow

### PR DESCRIPTION
**Pull request by sithforri**

### CODE REVIEW

**Review by cpb-gh — CHANGES_REQUESTED** [review]
one suggestion for a syntax fix and then we can merge.

File: functions/line_follow.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+from spike import PrimeHub, LightMatrix, Button, StatusLight, ForceSensor, MotionSensor, Speaker, ColorSensor, App, DistanceSensor, Motor, MotorPair
+from spike.control import wait_for_seconds, wait_until, Timer
+from math import *
+
+### FUNCTION START
+def get_speed( low_speed, high_speed, pct ):
+    return low_speed+((high_speed-low_speed)*pct)
+
+def ease(easetype, pct):
```
[the remark this answers is not in the document]
Comment (sithforri) [reply functions/line_follow.py:9]: yes placeholder

**Comment by sithforri** [comment]
fixed, please pull

**Review by cpb-gh — CHANGES_REQUESTED** [review]
two issues:
* a new function, turn_code.py has been added - should we add them both in this one PR?
* It looks like turn_code.py also has a get_speed() function and we should only have one of those total in the combined_functions file so only one of your files should contain this function, or name them differently if they should be different.

**Review by cpb-gh — APPROVED** [review]
all the fixes look good!

**Pull request merged**